feature/about/src/main/res/values/strings.xml

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="about_what_is_droidkaigi">What is DroidKaigi</string>
-    <string name="about_description_droidkaigi">DroidKaigiはエンジニアが主役のAndroidカンファレンスです。 \nAndroid技術情報の共有とコミュニケーションを目的に、2019年2月7日(木)、8日(金)の2日間開催します。</string>
-    <string name="about_access_to_place">会場アクセス</string>
-    <string name="about_check_map">Mapを開く</string>
-    <string name="about_staff_list">スタッフリスト</string>
-    <string name="about_check">見てみる</string>
-    <string name="about_privacy_policy">プライバシーポリシー</string>
-    <string name="about_license">ライセンス</string>
-    <string name="about_app_version">アプリバージョン</string>
+    <string name="about_what_is_droidkaigi">What is DroidKaigi?</string>
+    <string name="about_description_droidkaigi">DroidKaigi is a conference tailored for Android developers. It\'s scheduled to take place on the 7th and 8th of February 2019.</string>
+    <string name="about_access_to_place">Directions</string>
+    <string name="about_check_map">Open with Maps</string>
+    <string name="about_staff_list">Committee Members</string>
```

Comment:
maybe `DroidKaigi Members` is better? (`Committee Members` sounds very formal, so I'm not sure if I would click that button if I saw it)

Reply:
@panpanini Thanks for your following! 👍 
I think `DroidKaigi Members` should be works, but we(DroidKaigi) looks already using `committee` on the official website...?

![img](https://i.gyazo.com/f0cc8c6ad00209a33fc81a87d584ac6b.png)

Reply:
🤔 Okay, let's go with the word *Committee Members*! We do not need to rethink this word until other English speakers feel weird.